src/Middleware/RateLimiting/src/RateLimiterOptions.cs

```diff
@@ -11,30 +13,25 @@ namespace Microsoft.AspNetCore.RateLimiting;
 /// </summary>
 public sealed class RateLimiterOptions
 {
-    // TODO - Provide a default?
-    private PartitionedRateLimiter<HttpContext> _limiter = new NoLimiter<HttpContext>();
-    private Func<HttpContext, RateLimitLease, Task> _onRejected = (context, lease) =>
-    {
-        return Task.CompletedTask;
-    };
+    // Stores all of the keys for each partition so that we reuse the same objects.
+    internal IDictionary<string, ISet<DefaultKeyType>> _partitionKeys = new Dictionary<string, ISet<DefaultKeyType>>();
+
+    internal IDictionary<string, DefaultRateLimiterPolicy> PolicyMap { get; }
```

Comment:
Internal properties should use the concrete collection types not the interface unless needed.